### PR TITLE
META and debian changelog

### PR DESCRIPTION
**Pull request by toolslive**

### CODE REVIEW

File: META [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- conf -*-
 
 description = "Arakoon client"
-version = "1.6.8"
+version = "1.6.9"
```
Comment (NicolasT) [inline META:4]: This got me wondering: do we need new `arakoon-client` releases when only _node_ things changed?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,7 @@
+arakoon (1.6.9) unstable; urgency=low
+  * new upstream version
+
+ -- Romain Slootmaekers  <romain@incubaid.com>  Wed, 12 Feb 2014 16:00:00 +0100
```
Comment (NicolasT) [inline debian/changelog]: I guess the date will need to be adapted?

Comment (toolslive) [inline debian/changelog]: do we care?

Comment (domsj) [inline debian/changelog]: I like it better when it's correct, and it's only a small effort. (You can even do it straight from github.)
So in short: yes, I care.

**Pull request merged**